qa-pipelines/flags.yml

```diff
@@ -4,37 +4,39 @@
 # The settings in this file will also be valid for our most complete pipeline
 # (which performs an upgrade and all tests)
 
-# deploy cf for upgrade pipelines
-enable-cf-deploy-pre-upgrade: true
+# Report status as steps complete
+status-reporting: true
+
+# version to upgrade from; if null, upgrade is not performed.
+upgrade-from-version: scf-sle-2.15.2+cf3.6.0.0.gde1bd02f
+
+# When upgrading, deploy cf-usb (and an app using it) pre-upgrade, and ensure
+# that it remains working post-upgrade
+enable-cf-usb-upgrade-tests: true
 
 # run tests before an upgrade
 enable-cf-smoke-tests-pre-upgrade: true
 enable-cf-brain-tests-pre-upgrade: true
 
-# This should *only* ever be true when running a *full* pre-upgrade pipeline. The only reason to do this is
-# when you want to deploy a pipeline which uses a custom config URL instead of determining it from the
-# s3.scf-config-* resources. A typical upgrade pipeline only runs cats post-upgrade, and a typical deploy
-# pipeline has all the 'pre-upgrade' flags set to false
+# This should *only* ever be true when running a *full* pre-upgrade pipeline.
```

Comment:
With the change of the pre-upgrade deploy to `upgrade-from-version`, this isn't as clear. Actually, I don't think it even works that way because custom bundle deployments are run as normal deployments, but with the custom URL put into `enable-cf-deploy` right?

Reply:
In other words we may want to remove `enable-cf-acceptance-tests-pre-upgrade` altogether

Reply:
Removed in 2682d149dedca87a4db2b3624952b57f9ebf58a0